api/DedicatedWorkerGlobalScope.json

```diff
@@ -358,7 +358,8 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "69"
+              "version_added": "69",
+              "notes": "Not supported in nested Web Workers. See <a href='https://crbug.com/41483010'>bug 41483010</a>."
```

Comment:
Let's move this into a [`worker_support`](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/api.md#web-workers-worker_support) sub-feature.

Reply:
I' m not quite sure here, as the feature itself is worker support **_only_**.

It is quite unnecessary to add one extra [worker_support](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/api.md#web-workers-worker_support) sub-feature.

Reply:
Good point, I take my suggestion back.